templates/page/page.component.inc

```diff
@@ -73,6 +67,24 @@ function europa_preprocess_page(&$variables, $hook) {
   }
   $variables['site_header'] = array('#theme' => array('site_header'));
   $variables['messages'] = $variables['show_messages'] ? array('#theme' => array('status_messages')) : '';
+
+  $variables['page_header'] = array(
+    '#theme' => array('page_header'),
+    '#identity' =>  $variables['site_name'],
+    '#title_prefix' => $variables['title_prefix'],
+    '#title_suffix' => $variables['title_suffix'],
+    '#title' => drupal_get_title(),
+    '#introduction' => NULL,
+    '#meta' => array(),
```

Comment:
These two last lines is included to prepare the next developments. 
If you are not comfortable with these 'useless' lines, I can remove them.